src/server/routes/api/document/index.js

```diff
@@ -1259,12 +1260,30 @@ const emailRelatedPaperAuthors = async doc => {
 
   // TODO RPN use semantic search score etc. in future
   // just send to all for now, as we already have a small cutoff at 30
-  const paperIsGoodFit = async paper => true; // eslint-disable-line
+  const notReviewPaper = async paper => {
+    const reviewTypeUI = 'D016454';
+    const pubTypes = _.get( paper, ['pubmed', 'pubTypes'] );
+    const notReview = !_.find( pubTypes, ['UI', reviewTypeUI] );
+    return notReview;
+  };
+
+  const publishedNYearsAgo = ( paper, years = MAX_AGE_PAPER_YEARS ) => {
+    const dateNow = new Date();
+    const dateYearsAgo = ( new Date() ).setFullYear( dateNow.getFullYear() - years );
+    const paperDate = new Date( _.get( paper, ['pubmed', 'ISODate'] ) );
+    return isWithinInterval( paperDate, { start: dateYearsAgo, end: dateNow } );
+  };
+
+  const paperIsGoodFit = async paper => publishedNYearsAgo( paper ) && notReviewPaper( paper );
+
+  const byPaperDate = (a, b) => {
+    const getISODate = p => _.get( p, ['pubmed', 'ISODate'] );
+    return new Date( getISODate( a ) ) - new Date( getISODate( b ) );
+  };
 
   const getSendablePapers = async papers => {
     const isGoodFit = await Promise.all(papers.map(paperIsGoodFit));
-
-    return papers.filter((paper, i) => isGoodFit[i] && hasContact(paper));
+    return papers.filter((paper, i) => isGoodFit[i] && hasContact(paper)).sort( byPaperDate );
```

Comment:
The sort doesn't look strictly necessary, but it's probably useful for debugging.